### DISCUSSION TITLE
Why did they reduce the scroll size in the editor? Is there any way to increase it?

### DISCUSSION THREAD

**Discussion by SampaioPedroDev** [microsoft/vscode-discussions#2413] [Extension Development QnA] [answered]
As you can see in the image, we reduced the size of the scroll, it was very useful with the size it was, is there any way to increase it?

![Screenshot 2024-11-26 at 13 55 40](https://github.com/user-attachments/assets/deefbac6-4d60-4600-bfe5-aab3c35e855d)

**Comment by isidorn** [comment, marked as answer]
This is a user quesiton, and this forum is for VS Code Extension Authors.
Please ask this question on Stack Overflow
